### ISSUE TITLE
Sphere of Influence

### ISSUE THREAD

**Issue by AlteOgre** [VulcanForge/pvp-mode#84]
Spheres of influence (SoI) of factions can be used to force PvP Mode ON for any enemy player entering a faction's SoI. This may be a desirable feature for some servers.

SoI override zones are not that favourable though. Some SoI's of enemy factions overlap completely, which conflicts with the basic starting point that players should have full control over their PvP Mode and have the liberty to choose to play as a character in any of the factions. Example: Dorwinion's SoI is completely covered by Mordor's SoI, which would imply a player in Dorwinion will have PvP Mode forced ON within Dorwinion as soon as he killed a Mordor Orc attacking Dorwinrim in his neighborhood. Consequently, this feature will get a very low priority status, if ever implemented. This is to be treated in the same way as the PvP Mode forced ON override at night, which was desired by some server owners, but also conflicts with the mod's initial starting points on a fundamental level.

As mentioned in #62, I believe forced PvP Mode ON should either be defined per biome or per SoI, not both. Server can select which method, define alignment criterium, and only for the biome method either use the default list or customise it. Safe zones/biomes (#83) should work independent of the forced-ON zones, so safe zones should prevail over them.

**Comment by CraftedMods** [comment]
You're right. I didn't think about the possibility of overlapping spheres (rather circles). This makes this feature basically useless. But we can keep this issue here, maybe someone has an idea on how to make this work properly. To example, the biome could be taken into account too. There're some more or less neutral biomes - the Lone Lands, Wilderland and so on. So the part of Wilderland which is in the influence circle of Mordor forces enemies of Mordor to have PvP enabled. But people aligned with Dorwinion wouldn't be forced to have PvP enabled while they're in Dorwinion. This would allow a more fine grained biome to faction distribution - especially for "neutral" biomes. But this is something for the far future, I guess.
